gfsphysics/GFS_layer/GFS_typedefs.F90

```diff
@@ -4060,12 +4068,16 @@ subroutine control_initialize (Model, nlunit, fn_nml, me, master, &
     Model%ntrw             = get_tracer_index(Model%tracer_names, 'rainwat',    Model%me, Model%master, Model%debug)
     Model%ntsw             = get_tracer_index(Model%tracer_names, 'snowwat',    Model%me, Model%master, Model%debug)
     Model%ntgl             = get_tracer_index(Model%tracer_names, 'graupel',    Model%me, Model%master, Model%debug)
+    Model%nthl             = get_tracer_index(Model%tracer_names, 'hailwat',    Model%me, Model%master, Model%debug)
     Model%ntclamt          = get_tracer_index(Model%tracer_names, 'cld_amt',    Model%me, Model%master, Model%debug)
     Model%ntlnc            = get_tracer_index(Model%tracer_names, 'water_nc',   Model%me, Model%master, Model%debug)
     Model%ntinc            = get_tracer_index(Model%tracer_names, 'ice_nc',     Model%me, Model%master, Model%debug)
     Model%ntrnc            = get_tracer_index(Model%tracer_names, 'rain_nc',    Model%me, Model%master, Model%debug)
     Model%ntsnc            = get_tracer_index(Model%tracer_names, 'snow_nc',    Model%me, Model%master, Model%debug)
     Model%ntgnc            = get_tracer_index(Model%tracer_names, 'graupel_nc', Model%me, Model%master, Model%debug)
+    Model%nthnc            = get_tracer_index(Model%tracer_names, 'hail_nc',    Model%me, Model%master, Model%debug)
```

Comment:
As mentioned elsewhere in the comments, I'm assuming there are changes to a field_table file to use these new tracers?

Reply:
Yeah, there is a new field_table. Where in the tree would I commit that file?

Reply:
I don't think that the field_tables are in this or the parent (ufs-weather-model) repository anywhere. I think that they are considered as part of the input data and are typically staged on the platforms where regression tests are performed. You could just attach the field_table that you used as part of a comment in this PR so that the file can be staged when regression tests are done. @climbfuji, do you have a better idea?

Reply:
No, attaching it to the PR is a good idea, thanks.